src/libs/ReportUtils.ts

```diff
@@ -763,7 +791,7 @@ function isDraftReport(reportID: string | undefined): boolean {
  * Returns the report
  */
 function getReport(reportID: string): OnyxEntry<Report> {
```

Comment:
The utils method will use up to date data from Onyx in the moment they are called.
The main thing is not to use method like this inside component as a source of data to render, as it can lead to using outdated info.

Reply:
Discussed in real life, this method should not be exported. Lets not add new usages of this method outside of the ReportUtils and then in next PR substitute other places and add a test that makes sure we do not export it